packet_transport.go

```diff
@@ -0,0 +1,15 @@
+package swim
+
+// PacketTransportConfig is used to configure a udp transport
+type PacketTransportConfig struct {
+	// BindAddrs is representing list of address to use for  UDP communication
+	BindAddress string
```

Comment:
// BindAddrs is representing list of address to use for  UDP communication 

-> // BindAddrs is representing the address to use for  UDP communication